### DISCUSSION TITLE
Alternatives to github desktop

### DISCUSSION THREAD

**Discussion by bytebebop** [community/community#133883] [Other Feature Feedback, Questions, & Ideas] [answered]
I have been trying GitHub Desktop as a visual panel for my commits, I have investigated some other Sourcetree alternatives but I am not convinced by any of them. Based on your experience, do you recommend any?

**Comment by oscar-castaneda-dev** [comment, marked as answer]
I have used gitkraken for years, I recommend that tool, it is free for open source projects but if you want to manage repositories you have to pay

https://www.gitkraken.com/

**Reply by bytebebop** [reply]
I had heard about that platform but had not tried it, thanks for the recommendation.

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @bytebebop!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the New to GitHub category is solely related to conversations around getting started on using GitHub and the community for beginners of GitHub, Community on Discussions, and those new to different areas of the product alike! This question should be in the General category. I've gone ahead and moved it for you. Good luck!
